src/views/communities/index.tsx

```diff
@@ -47,190 +53,98 @@ function CommunitiesScreen() {
-                    <Select
-                        value={textCommunitiesOrder(communtiesOrder)}
-                        onPress={() => modalizeOrderRef.current?.open()}
-                    />
-                </View>
-                {/* {refreshing && (
-                    <ActivityIndicator
-                        style={{ marginBottom: 22 }}
-                        animating
-                        color={ipctColors.blueRibbon}
-                    />
-                )} */}
-            </>
-        );
-    };
-
-    const renderCommunityOrder = () => {
-        return (
-            <Modalize
-                ref={modalizeOrderRef}
-                HeaderComponent={renderHeader(
-                    i18n.t('generic.order'),
-                    modalizeOrderRef
-                )}
-                adjustToContentHeight
+    return (
+        <ScrollView>
+            <View
+                style={{
+                    marginHorizontal: 18,
+                    flexDirection: 'row',
+                    justifyContent: 'space-between',
+                    alignItems: 'center',
+                    marginBottom: 9,
+                }}
             >
-                <View
-                    style={{
-                        height: 110,
-                    }}
+                <Text style={styles.communityHeader}>
+                    {i18n.t('generic.communities')}
+                </Text>
+                <Pressable
+                    hitSlop={10}
+                    onPress={() => navigation.navigate(Screens.ListCommunities)}
                 >
-                    <RadioButton.Group
-                        onValueChange={(value) => handleChangeOrder(value)}
-                        value={communtiesOrder}
-                    >
-                        <RadioButton.Item
-                            key="bigger"
-                            label={i18n.t('generic.bigger')}
-                            value="bigger"
-                        />
-                        <RadioButton.Item
-                            key="nearest"
-                            label={i18n.t('generic.nearest')}
-                            value="nearest"
-                        />
-                    </RadioButton.Group>
-                </View>
-            </Modalize>
-        );
-    };
-
-    return (
-        <>
+                    <Text style={styles.viewAll}>
+                        {i18n.t('generic.viewAll')} ({communitiesCount})
+                    </Text>
+                </Pressable>
+            </View>
             <FlatList
+                // TODO: Although the useEffect limits the number of items to 5, I added slice to make sure if communities.length is greater than 5, it will show the first 5 items.
                 data={[
                     { publicId: 'for-compliance-sake-really' } as any,
-                ].concat(communities)}
+                ].concat(communities.slice(0, 5))}
```

Comment:
1. I'm confused. If the result has 5, how can it has more than five?
2. given then layout changed, no need to concat

Reply:
Already changed to control this on reducer instead. Fixed.